addons/flashlights/CfgVehicles.hpp

```diff
@@ -0,0 +1,45 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            //todo: add flashlight attach actions
+        }; 
+    };
+    
+    class NATO_Box_Base;
+    class EAST_Box_Base;
+    class IND_Box_Base;
+    class FIA_Box_Base_F;
+
```

Comment:
No. We are talking about the item entries. https://github.com/acemod/ACE3/blob/master/addons/medical/CfgVehicles.hpp#L746